lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -169,6 +246,52 @@ def process(historic_forecast, truth, distribution, cycletime, units=None,
             contains a coefficient_index dimension coordinate and a
             coefficient_name auxiliary coordinate.
     """
+    # Support the initial cycles when using a training dataset, where there
+    # might not yet be any available historic forecasts and truth to calibrate
+    # with. In this instance, only a warning is raised, as this would be
+    # expected behaviour when no training dataset exists.
+    if not any([historic_forecast, truth, combined]):
+        msg = ("In order to calculate the EMOS coefficients then either "
+               "the historic_forecast {} and the truth_filepath {} "
+               "should be specified, or the combined_filepath {} should be "
+               "specified alongside the historic_forecast_identifier {} and "
+               "truth_identifier {}. In this case the arguments provided "
+               "were not sufficient.".format(
+                    historic_forecast, truth, combined,
+                    historic_forecast_dict, truth_dict))
+        warnings.warn(msg)
+        return
+
+    if any([historic_forecast, truth]):
+        if all([historic_forecast, truth]):
+            if any([combined, historic_forecast_dict, truth_dict]):
+                msg = ("If the historic_filepath and truth_filepath arguments "
+                       "are specified then none of the the combined_filepath, "
+                       "historic_forecast_identifier and truth_identifier "
+                       "arguments should be specified.")
+                raise ValueError(msg)
+            else:
+                pass
+        else:
+            msg = ("Both the historic_filepath and truth_filepath arguments "
+                   "should be specified if one of these arguments are "
+                   "specified.")
+            raise ValueError(msg)
+
+    if any([combined, historic_forecast_dict, truth_dict]):
+        if all([combined, historic_forecast_dict, truth_dict]):
+            pass
+        else:
```

Comment:
Again I think this can be written as just an if not an if else.

Reply:
I've converted this into an `if not`.